cmd/server/assets/email/_plainheader.html

```diff
@@ -0,0 +1,9 @@
+{{- define "email/plainheader" -}}
+Subject: COVID-19 Verification Server Invitation
```

Comment:
Can be a follow up, but I think we want to say "Exposure Notifications", not "COVID"